src/frontend/src/components/public/PPQConnect/messages.ts

```diff
@@ -0,0 +1,27 @@
+export default {
+  Headings: {
+    Title: {
+      en: `Thank you for answering the PPQ`,
+    },
+    Subtitle: {
+      en: `Your answers will help us streamline the review of this PIA.`,
+    },
+    One: {
```

Comment:
just not sure if one two three is  a good naming practice here

Reply:
I agree. Are these Step One, Step two, etc? Then have an object called 'steps' and have the numbers nested One, Two Three.